src/matrix.jl

```diff
@@ -131,3 +132,30 @@ function split(A::IntervalMatrix{T}) where {T}
 
     return C, S
 end
+
+"""
+    rand(A::IntervalMatrix{T}; rng::AbstractRNG=GLOBAL_RNG) where {T}
+
+Sample a concrete matrix ``B`` from an interval matrix ``A`` such that every
+entry in ``B`` belongs to the corresponding interval in ``A``.
+
+### Input
+
+- `A`   -- interval matrix
+- `rng` -- (optional, default: `GLOBAL_RNG`) random-number generator
+
+### Output
+
+A concrete matrix.
+"""
+function rand(A::IntervalMatrix{T}; rng::AbstractRNG=GLOBAL_RNG) where {T}
+    m, n = size(A)
+    B = Matrix{T}(undef, m, n)
+    @inbounds for j in 1:n
+        for i in 1:m
+            itv = A[i, j]
+            B[i, j] = rand(Uniform(itv.left, itv.right))
```

Comment:
We can as well use `Base.random` here and not require `Distributions`, by letting `B[i, j] = (itv.right - itv.left)  * rand(rng) + itv.left`.

Reply:
Okay, I changed it.